packages/teleport-generator-shared/src/typings/generators.ts

```diff
@@ -137,13 +137,13 @@ export interface ProjectGeneratorOptions {
   skipValidation?: boolean
 }
 
-export type ProjectGeneratorFunction = (
-  uidl: ProjectUIDL,
-  options?: ProjectGeneratorOptions
-) => Promise<{
-  outputFolder: GeneratedFolder
-  assetsPath?: string
-}>
+// export type ProjectGeneratorFunction = (
```

Comment:
you can remove this interface, it is only used by the project exporters in the example folder and should not be here anyway